libnetwork/controller_linux.go

```diff
@@ -20,7 +20,12 @@ func (c *Controller) FirewallBackend() *system.FirewallInfo {
 		return nil
 	}
 	if usingFirewalld {
-		return &system.FirewallInfo{Driver: "iptables+firewalld"}
+		info := &system.FirewallInfo{Driver: "iptables+firewalld"}
+		reloadedAt := iptables.FirewalldReloadedAt()
+		if !reloadedAt.IsZero() {
+			info.Info = append(info.Info, [2]string{"ReloadedAt", reloadedAt.String()})
```

Comment:
We should probably use `RFC3339` here, as we tend to (should!) do for values returned by the API;

For systemtime, we include nano-seconds, but perhaps that's a bit too precise?

```suggestion
			info.Info = append(info.Info, [2]string{"ReloadedAt", reloadedAt.Format(time.RFC3339Nano)})
```

If so, we could just use without;

```suggestion
			info.Info = append(info.Info, [2]string{"ReloadedAt", reloadedAt.Format(time.RFC3339)})
```